libc/src/__support/FPUtil/FPBits.h

```diff
@@ -85,9 +85,9 @@ template <> struct FPBaseProperties<FPType::X86_Binary80> {
 } // namespace internal
 
 template <FPType fp_type>
-struct FPProperties : public internal::FPBaseProperties<fp_type> {
+struct FPBaseMasksAndShifts : public internal::FPBaseAttr<fp_type> {
```

Comment:
Doc ? "`FPBaseMasksAndShifts` derives useful constants from the `FPBaseAttr` layout"